README.md

```diff
@@ -106,3 +106,18 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
 
 * Fry
 * Kestrel
+
+### .sh file instructions
```

Comment:
```suggestion
<details>
     <summary>## Included shell scripts (Click to expand)</summary>
```
I suggest that we keep this heading on the same level as "Cluster support" so that it doesn't seem like a subheading to the previous one. Also I changed the wording to (1) start the heading with a capital letter and (2) give an explanation of what they are vs what extension they have. "Instructions for included shell scripts" could also work but is a little wordy... 🤔 

I think I also like the idea of moving the collapsible part here?